### PR TITLE
OSDOCS-2883: default balancing algorithm leastconn

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2883

Preview: https://deploy-preview-38403--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

### CODE REVIEW

**Comment by bmcelvee** [comment]
@frobware and @quarterpin would you mind reviewing this PR, please? Thanks!

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
 [cols="3*", options="header"]
 |===
 |Variable | Description | Environment variable used as default
-|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
+|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`. The default value is `random`.| `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
```
Comment (quarterpin) [inline modules/nw-route-specific-annotations.adoc:22]: If no annotation is added to the route, the route will have `leastconn` as the default which in turns comes via  `ROUTER_LOAD_BALANCE_ALGORITHM` proxy pod variable.  Hence the statement should be re-phrased to reflect the same.

Comment (quarterpin) [reply modules/nw-route-specific-annotations.adoc]: One correction is needed: `The default value is leastconn`.  The rest looks good.

File: modules/nw-route-specific-annotations.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
 [cols="3*", options="header"]
 |===
 |Variable | Description | Environment variable used as default
-|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
+|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`.  The default value is `random`.| `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
```
Comment (quarterpin) [inline modules/nw-route-specific-annotations.adoc:22]: One correction to the statement: `The default value is leastconn`

Comment (bmcelvee) [reply modules/nw-route-specific-annotations.adoc:22]: Thanks, @quarterpin! When I updated the PR yesterday I didn't save the change. :woman_facepalming: Updating now.

**Comment by mjpytlak** [comment]
Peer review LGTM.

**Comment by bmcelvee** [comment]
I think we are safe to proceed with merging and cherry picking. If any additional changes are needed, I will open a follow-up PR.

**Pull request merged**